src/renderer.ts

```diff
@@ -226,6 +226,8 @@ export const render = async ({ data, width, height, scrobbles, artists }: Render
   for (let i = 0; i < artistsData.length; i++) {
     const artist = artistsData[i]
 
+    if (!artist.image) continue
```

Comment:
oh, so some artists dont even have avatars? interesting, i might add some default avatar rendering scenarios for these.

as for the code, i'd use brackets here:
```js
    if (!artist.image) {
      continue
    }
```

Reply:
tbh, if artist don't have image, then with my solution it will just skip artist name. it will be good if it is done so that if there is no avatar, then the artist's name is shifted to the left (to the place where the artist's avatar should be)

Reply:
yeah, i will probably make a handler for this case. but for now, we can stick with this solution!
you may also add `// TODO: handle when there's no avatar` comment here!

Reply:
also, could you please give an example of an artist without image? so i can properly test it

Reply:
this is probably related to artists, that haven't registered to spotify, but added as another artist in track, idk how exactly it works.
for example track Pendulum, Liam Howlett (with this artist I have found this error) - Immunize. `image` is just `undefined`

Reply:
@bit0r1n added a fallback render scenario in https://github.com/nitreojs/vk-cover-spotify-playback/commit/4fff474d6edb94ba277b4b273820c0c4ebb62f61